### PR TITLE
Print reserved instances that have retired in past week

### PR DESCRIPTION
**Pull request by emmasax1**
Description and Impact
----------------------
Print the expiration date of recently retired reserved instances to help users complete their audits.

Deploy Plan
-----------
* checkout master
* `op accept-pull 16`
* `soyuz deploy`

Rollback Plan
-------------
* To roll back this change, revert the merge with: `git revert -m 1 MERGE_SHA` and perform another deploy.

URLs
----
http://docs.aws.amazon.com/sdkforruby/api/Aws/EC2/Client.html

QA Plan
-------
- [x] Run `GLI_DEBUG=true bin/sport-ngin-aws-auditor audit [account]` from this branch of the repo and watch it also print the recently (within 1 week) retired reserved instances
- [x] Run `GLI_DEBUG=true bin/sport-ngin-aws-auditor audit -s [account]` from this branch of the repo and watch this also work when printing to slack channel

### CODE REVIEW

**Comment by emmasax1** [comment]
### QA:

#### In the terminal:
screen shot 2016-08-25 at 9 27 20 am

#### In Slack:
screen shot 2016-08-25 at 9 28 35 am

File: lib/sport_ngin_aws_auditor/instance_helper.rb [hunk 1]
```diff
@@ -34,15 +34,23 @@ def compare(tag_name)
...
         differences[key] = ris_count - instance_count
       end
+      
       add_instances_with_tag_to_hash(instances_with_tag, differences)
       differences
     end
 
+    def get_recent_retired_reserved_instances
+      get_retired_reserved_instances.select do |ri|
+        ri.expiration_date > (Time.now - 604800)
```
Comment (anfleene) [inline lib/sport_ngin_aws_auditor/instance_helper.rb]: Maybe make a comment about how these are the RIs that expired in the last week.

File: lib/sport_ngin_aws_auditor/scripts/audit.rb [hunk 2]
```diff
@@ -52,26 +54,36 @@ def self.gather_data(class_type, tag_name)
...
         compared_array = []
+
         instance_hash.each do |key, value|
           compared_array.push(Instance.new(key, value))
         end
-        compared_array
+
+        return_array.push(compared_array)
+        return_array.push(retired_reserved_instances) if retired_reserved_instances
+        return_array
```
Comment (anfleene) [inline lib/sport_ngin_aws_auditor/scripts/audit.rb]: you could do this in one line if you do `[comared_array, retired_reserved_instances]`

**Comment by anfleene** [comment]
@emmasax1 Seems like you duplicated the same code like 3 times. We can leave it for this PR but I think at this point we probably have enough code to spot a nice refactor.

**Comment by anfleene** [comment]
CR looks good 👍

**Pull request merged**